content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -106,3 +106,56 @@ Once that option is enabled, go back to the :guilabel:`Profile` options page, an
 
 Once all those steps have been taken, perform a test scan to ensure the Zebra scanner is working
 properly, as intended.
+
+Honeywell mobile computer scanner
+=================================
+
+When using Honeywell scanners, follow the instructions below to ensure barcodes can be scanned in
+Odoo.
+
+Begin on the Honeywell scanner's home screen, and select :guilabel:`Settings`, represented by a
+:guilabel:`⚙️ (gear)` icon. Then, click :guilabel:`Honeywell Settings`, followed by
+:guilabel:`Scanning`.
+
+From here, click :guilabel:`Internal Scanner`, followed by :guilabel:`Default Profile`. From the
+resulting list of options, select :guilabel:`Data Processing Settings`.
+
+The :guilabel:`Data Processing Settings` specify how barcode data is processed by the computer.
+Locate the :guilabel:`Wedge Method` setting. By default, this is set to :guilabel:`Standard`. Change
+the :guilabel:`Wedge Method` setting to :guilabel:`Keyboard`.
+
+.. image:: hardware/hardware-honeywell-settings.png
+   :align: center
+   :alt: Data processing settings options for Honeywell scanner.
```

Comment:
```suggestion
Locate the :guilabel:`Wedge Method` setting. By default, this is set to :guilabel:`Standard`.

.. image:: hardware/hardware-honeywell-settings.png
   :align: center
   :alt: Data processing settings options for Honeywell scanner.

Change the :guilabel:`Wedge Method` setting to :guilabel:`Keyboard`.
```
OPTIONAL: Since the screenshot you used *still* shows the 'Standard' setting -- it feels like the placement of the image should come *before* mentioning that it needs to be switched to 'Keyboard.'

With this ordering, the reader will be able to follow along just a little bit easier. At least, that's how I *hope* they'd feel! haha what are your thoughts?

Reply:
Yeah, I totally get that logic. Because I had to rely on user manuals for some of these my options were pretty limited 😅 But I agree that changing the placement could help avoid confusion, so I'll change it.

Suggestion implemented ✅